packages/pdfrx/lib/src/widgets/pdf_viewer_params.dart

```diff
@@ -1018,6 +1045,9 @@ class PdfViewerSelectionMagnifierParams {
     this.builder,
     this.shouldBeShownForAnchor,
     this.maxImageBytesCachedOnMemory = defaultMaxImageBytesCachedOnMemory,
+    this.calcPosition,
+    this.shouldShowMagnifier,
```

Comment:
The inconsistency between existing `shouldBeShownForAnchor` and `shouldShowMagnifier` naming rules. I'm thinking renaming of the existing `shouldBeShownForAnchor`. On the other hand, the problem is that they are very similar and confusing. We should do something on that (at least, explain the difference on document)...